src/main/java/domain/player/Name.java

```diff
@@ -0,0 +1,40 @@
+package domain.player;
+
+public class Name {
+    private final String value;
+    private final static int MAXIUM_NAME_LENGTH = 5;
```

Comment:
```suggestion
    private final static int MAXIMUM_NAME_LENGTH = 5;
    
    private final String value;
```
1. MAXIMUM 오타가 있어요.
2. 자바 컨벤션은 상수가 가장 위에 배치되어요.